_scicomputing/store_objectstore.md

```diff
@@ -37,34 +38,52 @@ An additional tool is [Motuz](http://motuz.fredhutch.org), a tool that facilitat
+Once you have [obtained S3 credentials](/scicomputing/access_credentials/), you can use them to transfer files from/to the PI S3 buckets.
 
-#### Credentials and Permissions
+By default, everyone in the lab with an AWS username will have access to all data in all buckets, however the prefixes `readonly/` and `SR/` are typically restricted.  Data Managers and Admins have full permissions on all files in all buckets.
+
+It is possible to create additional restrictions if necessary.  For example, if a lab needs to store confidential PHI data such as for a clinical trial and only a certain subset of users in the lab are authorized to access that data, a specific prefix can be created that restricts access to only those authorized users.  Please submit a helpdesk ticket for assistance in setting this up.
+
+#### Backup and Retention
 
-Once you have [obtained S3 credentials](/scicomputing/access_credentials/), you can use them to transfer files from/to the PI S3 buckets. If you work in the lab of the PI Jane Doe, your lab's S3 bucket name will be `fh-pi-doe-j-eco` for the private bucket and `fh-pi-doe-j-eco-public` for the public bucket. Please substitute your lab's actual bucket name when using the examples in our Resource Library demos.
+Data stored in managed S3 buckets are not backed up in the traditional sense, but rather versioned: if a new version of a file is uploaded, the older version is saved in S3.  Similarly, if data is deleted, the versions aren't and can be retrieved for up to 60 days.
 
-By default, everyone in the lab with an AWS username will have access to all data in all buckets, however the prefixes `readonly/` and `SR/` are typically restricted.  Data Managers and Admins have full permissions on all files in all buckets.  While this default setup works well for most labs, it is possible to create additional restrictions as well if necessary.  For example, if a lab needs to store confidential PHI data such as for a clinical trial and only a certain subset of users in the lab are authorized to access that data, a specific prefix can be created that restricts access to only those authorized users.  Please submit a helpdesk ticket for assistance in setting this up.
+#### Security
+
+The Fred Hutch managed PI buckets in S3 (`fh-pi-doe-j-eco` and `fh-pi-doe-j-eco-public`) are appropriate for storage of restricted data, including PHI.
+
+#### Storage Tiering and Retrieval
+
+All managed S3 buckets (`fh-pi-doe-j-eco` and `fh-pi-doe-j-eco-public`) have Intelligent Tiering enabled. This means that data can be automatically moved to a different AWS S3 storage class after it has not been accessed for a certain period of time.  All files are still retrievable at any time and the URL for accessing this data does not change.
```

Comment:
Folks will be confused with the word "moved" here re: storage class even though you explain it.  I guarantee it!  hahah.  Perhaps we could say "labeled as a different AWS S3 storage class (which impacts how costs are calculated but doesn't change anything noticeable from our users' perspective)"???

Reply:
Yeah, it's a tricky term, but I like your approach.  Updated in c5215e6